src/components/plots/plot.ts

```diff
@@ -270,24 +270,25 @@ module Plottable {
       this._scales().forEach((scale) => scale._autoDomainIfAutomaticMode());
     }
 
-    private _updateExtentsForAttr(attr: string) {
+    protected _updateExtentsForAttr(attr: string, filter?: _Accessor) {
       var binding = this._attrBindings.get(attr);
       var accessor = binding.accessor;
-      var scale = binding.scale;
-      var coercer = (scale != null) ? scale._typeCoercer : (d: any) => d;
       var extents = this._datasetKeysInOrder.map((key) => {
         var plotDatasetKey = this._key2PlotDatasetKey.get(key);
         var dataset = plotDatasetKey.dataset;
         var plotMetadata = plotDatasetKey.plotMetadata;
-        return this._computeExtent(dataset, accessor, coercer, plotMetadata);
+        return this._computeExtent(dataset, accessor, plotMetadata, filter);
       });
       this._attrExtents.set(attr, extents);
     }
 
-    private _computeExtent(dataset: Dataset, accessor: _Accessor, typeCoercer: (d: any) => any, plotMetadata: any): any[] {
+    private _computeExtent(dataset: Dataset, accessor: _Accessor, plotMetadata: any, filter?: _Accessor): any[] {
       var data = dataset.data();
+      if (filter) {
```

Comment:
`filter != null`

Just noting that I'm starting to align more to the explicit `filter != null` rather than just the `filter` mainly due to a more stricter typing

For example, `projectorsReady()` used to have an implementation that looked like...

```typescript
return this._projections["x"] && this._projections["y"];
```

which meant that `projectionsReady()` had a return type of... `_Projection`... not `boolean`...